spec.html

```diff
@@ -24630,7 +24630,7 @@ <h1>Object.prototype.toLocaleString ( [ _reserved1_ [ , _reserved2_ ] ] )</h1>
         </emu-alg>
         <p>The optional parameters to this function are not used but are intended to correspond to the parameter pattern used by ECMA-402 `toLocaleString` functions. Implementations that do not include ECMA-402 support must not use those parameter positions for other purposes.</p>
         <emu-note>
-          <p>This function provides a generic `toLocaleString` implementation for objects that have no locale-specific `toString` behaviour. `Array`, `Number`, `Date`, and `Typed Arrays` provide their own locale-sensitive `toLocaleString` methods.</p>
+          <p>This function provides a generic `toLocaleString` implementation for objects that have no locale-specific `toString` behaviour. `Array`, `Number`, `Date`, and %TypedArray% provide their own locale-sensitive `toLocaleString` methods.</p>
```

Comment:
i think technically this would be `%TypedArray_prototype%`?

Reply:
I think 'provide' here is meant to mean "on the prototype" though? Because it certainly is `%TypedArray%.prototype.toLocaleString`, but that's also true for `Array`, `Number`, and `Date`.

Reply:
ah true, fair enough :-)